vm-setup/roles/firewall/tasks/iptables.yaml

```diff
@@ -110,13 +110,13 @@
         state: "{{ firewall_rule_state }}"
       loop: "{{ provisioning_host_ports }}"
 
-    - name: "iptables: Allow access to provisioning network from kind"
+    - name: "iptables: Allow access to bare metal provisioner network from kind"
```

Comment:
We are still keeping provisioning network name ?

Reply:
Yes, eventually there will be Provisioner and  Provisioning networks.  But in this specifc case of "{{ provisioning_host_ports }}" I have to check I might have missed this.

Reply:
I have changed "provisioning_host_ports" to "vm_host_ports"